highfidelity/hifi-content
# Meowing Cat

This script relies on having cat models and animations in the domain asset browser. The basic code allows you to add meow.js to a cat model to play a meow sound when it's clicked on. 

**Test Plan**
1. Go to hifi://studio/-198.2,28.4,-81.6
2. Click on the small white cat that is sitting on the chair
3. Confirm that the kitten meows and plays a meowing animation 

Note: the animations used currently are in an unfinished state, but shouldn't block QA.
